src/components/Battles.test.ts

```diff
@@ -0,0 +1,115 @@
+import { expect } from "chai";
+import { Clock } from "lolex";
+import { MenuGraphr } from "menugraphr";
+import { FullScreenPokemon } from "..";
+import { createBattleMover } from "../creators/createBattleMover";
+import { stubBlankGame } from "../fakes.test";
+import { IBattleTeam, IPartialBattleOptions, IPokemon } from "./Battles";
+import { IPlayer } from "./Things";
+
+const createGame = () => {
+    const { clock, fsp, player } = stubBlankGame(
+        {height: 1000,
+         width: 1000,
+         automaticallyAdvanceMenus: true},
+    );
+    const charmander = fsp.equations.newPokemon({
+        level: 99,
+        title: "CHARMANDER".split(""),
+        moves: [
+            {
+                remaining: 10,
+                title: "Scratch",
+                uses: 10,
+            },
+        ],
+    });
+
+    const enemyPokemon: IPokemon = fsp.equations.newPokemon({
+        level: 3,
+        title: "PIDGEY".split(""),
+        moves: [
+            {
+                remaining: 10,
+                title: "Growl",
+                uses: 10,
+            },
+        ],
+    });
+
+    return {clock, fsp, player, charmander, enemyPokemon};
+};
+
+const processBattle = (fsp: FullScreenPokemon, enemyPokemon: IPokemon, player: IPlayer, clock: Clock) => {
+    fsp.battles.startBattle({
+        teams: {
+            opponent: {
+                actors: [enemyPokemon],
+            },
+        },
+        texts: {
+            start: (team: IBattleTeam): string =>
+                `Wild ${team.selectedActor.nickname.join("")} appeared!`,
+        },
+    });
+    fsp.inputs.keyDownA(player);
+};
+
+describe("Battles", () => {
+    it("Ensures a status condition pokeball sprite is displayed when pokemon is affected by a status condition", (): void => {
```

Comment:
Since these are made to read like plain English, it's odd having an uppercase letter to start. Phrasing is a little off too.

Instead of:

> Battles Ensures a status condition pokeball sprite is displayed when pokemon is affected by a status condition

Use something like:

> Battles display a status condition Pokeball sprite when a Pokemon is affected by a status condition

Here and elsewhere.